src/api/helpers/get-ids-by-reference-number/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Application, Context } from '.keystone/types'; // eslint-disable-line
+
+/**
+ * getIdsByReferenceNumber
+ * get ids of application sections by application reference number
+ * returns application ids or if not found, returns null
+ * @param {Number} referenceNumber
+ * @param {Context} context
+ * @returns {Application} application ids or null
+ */
+const getIdsByReferenceNumber = async (referenceNumber: number, context: Context): Application => {
```

Comment:
Finding this naming confusing, can it just be `getApplicationByReferenceNumber`?

`Application` is not specifically for IDs.

To me it currently implies that it's getting multiple application IDs.